xml/depl_nostack_services.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE cha.depl.nostack PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha.depl.ostack">
+  <title>Deploying the Non-&ostack; Components</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+    <dm:maintainer>fs</dm:maintainer>
+    <dm:status>editing</dm:status>
+    <dm:deadline/>
+    <dm:priority/>
+    <dm:translation>no</dm:translation>
+    <dm:languages/>
+  </dm:docmanager>
+ </info>
+  <para>
+    In addition to &ostack; &barcl;s, &suse; &ostack; Cloud includes several components that can be configured using the appropriate  &crow; &barcl;s.
+   </para>
+    <sect1 xml:id="sec.depl.nostack.ntp">
+  <title>Configuring the NTP Service</title>
+  <para>
+    The NTP service is responsible for keeping the clocks in your cloud servers
+    in sync. The NTP component is deployed on the Administration Server automatically using the default settings. The NTP &barcl; can be used to specify IP addresses of the external NTP servers and assign specific roles to the desired nodes. The following parameter can be configured using the NTP &barcl;:
+  </para>
+  <variablelist>
+    <varlistentry>
+      <term>External servers</term>
+      <listitem>
+	<para>
+	  A comma-separated list of IP addresses of external NTP servers.
+	</para>
+      </listitem>
+    </varlistentry>
+  </variablelist>
+  <para>
+    <!-- dpopov 2017-03-06: DEVs, please review the NTP barclamp chapter,
+	 especially the roles descriptions. -->
+    The NTP service consists of two different roles:
+  </para>
+  <variablelist>
+   <varlistentry>
+    <term><guimenu>ntp-server</guimenu>
+    </term>
+    <listitem>
+     <para>
+      A node that acts as an NTP server for NTP clients in your cloud. There
+      can be more than one node with the ntp-server role in your cloud. In
+      this scenario, the NTP server nodes can communicate with each other to
+      keep their time in sync.
```

Comment:
@bmwiedemann Thank you very much for your feedback! I'll update the text as per your suggestions.